client/my-sites/customer-home/cards/tasks/webinar/index.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Task from '../task';
+import temporaryIllustration from 'assets/images/customer-home/illustration--task-connect-social-accounts.svg';
+
+const Webinar = () => {
+	const translate = useTranslate();
+
+	return (
+		<Task
+			title={ translate( 'Get hands-on learning' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Expert instruction. Hands-on learning.' )` *ES Score: 6*